setup.py

```diff
@@ -264,6 +264,7 @@
     'obspy.plugin.waveform.AH': [
         'isFormat = obspy.io.ah.core:_is_ah',
         'readFormat = obspy.io.ah.core:_read_ah',
+        'writeFormat = obspy.io.ah.core:_write_ah1'
```

Comment:
Renaming the write part to AH1 would complicate our roundhouse test (read 'AH' but needs to write 'AH1').. maybe it can stay this way and if ever somebody decides to implement the other one we just have 'AH' and 'AH2'? But whatever you guys decide on is fine with me..